pom.xml

```diff
@@ -5,30 +5,34 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.7.1</version>
+        <version>2.7.6</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>com.mastercard.installments</groupId>
-    <artifactId>installments-bnpl-api-reference-app</artifactId>
+    <artifactId>bnpl-api-reference-application</artifactId>
```

Comment:
Why we are renaming the artifact id? 

This was previously set based on feedback.